CONTRIBUTING.md

```diff
@@ -242,9 +243,9 @@ If you need to restart your Julia session, just start at step 2 above.
 built and incorporate them automatically. You only need to rebuild
 Julia if you made code-changes that Revise cannot handle.
 
-For convenience, there are also `test-revise-*` targets for every `test-*`
-target that use Revise to load any modifications to Base into the current
-process before running the corresponding test. This can be useful as a shortcut
+For convenience, there are also `test-revise-*` targets for every [`test-*`
+target](https://github.com/JuliaLang/julia/blob/master/CONTRIBUTING.md#running-specific-tests) that use Revise to load any modifications to Base into the current
+system image before running the corresponding test. This can be useful as a shortcut
```

Comment:
This makes it sound like the `*.so` file on disk is modified.

Reply:
Good point. I was looking for something more specific than "process", but maybe that's the right word to begin with.

The system image is the serialization of ______ to disk. The state of the julia runtime?